src/main/java/frc/robot/RobotContainer.java

```diff
@@ -58,47 +52,61 @@ public RobotContainer() {
-                && DriverStation.getAlliance().get() == DriverStation.Alliance.Red,
-        m_drive);
-    Pathfinding.setPathfinder(new LocalADStarAK());
-    PathPlannerLogging.setLogActivePathCallback(
-        (activePath) ->
-            Logger.recordOutput("Odometry/Trajectory", activePath.toArray(new Pose2d[0])));
-    PathPlannerLogging.setLogTargetPoseCallback(
-        (targetPose) -> Logger.recordOutput("Odometry/TrajectorySetpoint", targetPose));
-
-    autoChooser =
-        new LoggedDashboardChooser<>(
-            "Auto Choices",
-            PathPlannerUtil.configureChooserWithPaths(AutoBuilder.buildAutoChooser()));
+    configureAutos();
 
     if (Constants.TUNING_MODE) {
-      // Set up FF characterization routines
-      autoChooser.addOption(
-          "DriveBase FF Characterization",
-          new FeedForwardCharacterization(
-              m_drive, m_drive::runCharacterizationVolts, m_drive::getCharacterizationVelocity));
+      configureTunableParameters();
     }
 
-    // Configure the button bindings
     configureButtonBindings();
   }
 
+  private void configureAutos() {
+    m_autoChooser.addDefaultOption("None", Commands.none());
+  }
+
+  private void configureTunableParameters() {
+    m_autoChooser.addOption(
+        "DriveDynamicForward", m_drive.characterizeDriveDynamic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "DriveDynamicReverse", m_drive.characterizeDriveDynamic(SysIdRoutine.Direction.kReverse));
+    m_autoChooser.addOption(
+        "DriveQuasistaticForward",
+        m_drive.characterizeDriveQuasistatic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "DriveQuasistaticReverse",
+        m_drive.characterizeDriveQuasistatic(SysIdRoutine.Direction.kReverse));
+    m_autoChooser.addOption(
+        "ErectorDynamicForward",
+        m_shooter.characterizeErectorDynamic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "ErectorDynamicReverse",
+        m_shooter.characterizeErectorDynamic(SysIdRoutine.Direction.kReverse));
+    m_autoChooser.addOption(
+        "ErectorQuasistaticForward",
+        m_shooter.characterizeErectorQuasistatic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "ErectorQuasistaticReverse",
+        m_shooter.characterizeErectorQuasistatic(SysIdRoutine.Direction.kReverse));
+    m_autoChooser.addOption(
+        "ShooterDynamicForward",
+        m_shooter.characterizeShooterDynamic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "ShooterDynamicReverse",
+        m_shooter.characterizeShooterDynamic(SysIdRoutine.Direction.kReverse));
+    m_autoChooser.addOption(
+        "ShooterQuasistaticForward",
+        m_shooter.characterizeShooterQuasistatic(SysIdRoutine.Direction.kForward));
+    m_autoChooser.addOption(
+        "ShooterQuasistaticReverse",
+        m_shooter.characterizeShooterQuasistatic(SysIdRoutine.Direction.kReverse));
```

Comment:
This selector is gonna get larger, it might make sense to make a FF Characterization Selector that gets polled when test mode is activated